automatic/chromium/tools/chocolateyInstall.ps1

```diff
@@ -14,7 +20,7 @@ $validExitCodes = @(0)
   if (Test-Path $Chromium) {
     $silentArgs = ''
   } else {
-    $silentArgs = '--system-level --do-not-launch-chrome'
+    $silentArgs = '--system-level'
   }
-  
-    Install-ChocolateyPackage $packageName $fileType $silentArgs $url $validExitCodes $checksum $checksumType
+
+    Install-ChocolateyPackage $packageArgs

```

Comment:
To do this requires splatting, or `@packageArgs` instead of `$packageArgs`. That means it will take `$packageArgs and split it by name and value and pass those things. So `$packageArgs[fileType]` becomes `-fileType 'exe'` as it is passed to the method.